src/main/java/seedu/event/DateTime.java

```diff
@@ -33,15 +37,15 @@ public LocalDateTime getDateTime() {
         try {
             return LocalDateTime.parse(dateTime, DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm"));
         } catch (DateTimeParseException e) {
-            if (!isAutoSet) {
-                UI.display("Unknown DateTime format provided. DateTime is set to local time in the format: "
-                        + "yyyy-MM-dd HHmm.");
-            }
-            isAutoSet = true;
-            return LocalDateTime.now();
+            isInvalid = true;
```

Comment:
Why not this version?
```
if (!isAutoSet) {
    UI.display("Unknown DateTime format provided. DateTime is set to local time in the format: "
                + "yyyy-MM-dd HHmm.");
    isAutoSet = true;
}
```

Reply:
Need another variable to check whether the parsing is done right, and if the error message for the wrong parsing is required, check EventList#add and EventList#editDateTime. In this case, null Localdatetime is returned when datetimeparseexception encountered.